pkg/archive/time.go

```diff
@@ -0,0 +1,12 @@
+package archive // import "github.com/docker/docker/pkg/archive"
```

Comment:
If you need to update; can you remove the `// import "` from this one? It's a new file, and we'll have to remove these before migrating to a separate module, so we may already remove here to reduce churn.